### PR TITLE
Adds handshaking and pulling by hands

### PR DESCRIPTION
**Pull request by Shadowtail117**
## About the Pull Request

Adds handshaking by targeting someone's hands on help intent when they are standing, and pulling people by hands if you are targeting them. In both cases, you have to be targeting a hand that they have, otherwise it will default to the base behavior.

image

## Why It's Good For The Game

Adds variety, and I think hugging being the only positive physical interaction you can do seems a bit weird.

## Did you test it?

Yes, if the image above wasn't clear >_>

## Changelog

:cl:
rscadd: You can now pull someone by the hands if you target a hand that they have and pull them.
rscadd: You can now shake someone's hand if you target a hand that they have and click on them with help intent.
/:cl:

### CODE REVIEW

File: code/modules/species/species.dm [hunk 1]
```diff
@@ -412,8 +412,22 @@ The slots that you can use are found in items_clothing.dm and are the inventory
...
+		switch(H.zone_sel.selecting)
+			if(BP_R_HAND)
+				if(U.has_organ(BP_R_HAND))
+					H.visible_message(SPAN_NOTICE("\The [H] shakes \the [U]'s hand."), SPAN_NOTICE("You shake \the [U]'s hand."))
+					zonefound = TRUE
+			if(BP_L_HAND)
+				if(U.has_organ(BP_L_HAND))
+					H.visible_message(SPAN_NOTICE("\The [H] shakes \the [U]'s hand."), SPAN_NOTICE("You shake \the [U]'s hand."))
+					zonefound = TRUE
+			//here is where tail entwining will go one day
```
Comment (Ilysen) [inline code/modules/species/species.dm:427]: Please remove this comment. It has the subtlety of a brick, and if it gets added later, we can worry about it then. :v

**Review by Ilysen — DISMISSED** [review]
Sky expressed some concern about this PR. I'm personally ambivalent to it (I like the hand-shaking thing), but if you actually want this, could you please update the title and body of the PR to reflect that it's not solely for a meme?

File: code/modules/species/species.dm (hunk 1 above)
Comment (Shadowtail117) [reply code/modules/species/species.dm:427]: nuuuuu

Comment (Ilysen) [reply code/modules/species/species.dm:427]: Yes, please. I can't tell if this is a meme PR or not as it is, and this comment and your response makes me feel like it probably is.

Comment (Shadowtail117) [reply code/modules/species/species.dm:427]: The underlying content is not a joke, I'm just capitalizing on the "hand holding illegal" meme. I'll remove it.

**Comment by Shadowtail117** [comment]
Done. Long live the meme :pensive:

**Comment by Ilysen** [comment]
I appreciate it. Code looks fine, but I'm ambivalent to the content. I'd like to defer to @Cupax3 here to make the final choice, since some other staff had expressed concerns, even though I don't feel like it's a problem in its current state.

**Comment by SkyCinnamon** [comment]
> I appreciate it. Code looks fine, but I'm ambivalent to the content. I'd like to defer to @Cupax3 here to make the final choice, since some other staff had expressed concerns, even though I don't feel like it's a problem in its current state.

my concerns have been addressed / I dont care as much anymore from my original post.

**Review by Cupax3 — APPROVED** [review]
LGTM

**Pull request merged**